README.md

```diff
@@ -25,3 +25,7 @@ _The Conventional Commit template before adding content, viewed in Vim editor_
 <img width="703" alt="Screen Shot 2023-01-28 at 12 53 16 PM" src="https://user-images.githubusercontent.com/12022922/215283102-a989033d-1ef6-4c88-be1d-24eed49ba618.png">
 
 _The Conventional Commit template after being filled out with all mandatory information_
+
+## Running Locally
+
+No idea what to write here, please help? 
```

Comment:
Just need to add the command for running the application locally. And it should be formatted as a code snippet, so prefixed and postfixed with three back ticks (```). 

```
npm run start
```

Reply:
Added. Let me know if I missed any steps to get it back up here.